Modules/Role/Http/Services/RoleService.php

```diff
@@ -2,38 +2,30 @@
 
 namespace Modules\Role\Http\Services;
 
-use Illuminate\Http\Request;
-use Illuminate\Support\Str;
-use Illuminate\Support\Facades\Auth;
-use Modules\Role\Entities\Role;
-use Modules\Role\Entities\Page;
-use Modules\Login\Entities\User;
-use Illuminate\Support\Facades\Hash;
 
+use Spatie\Permission\Models\Role;
+use Spatie\Permission\Models\Permission;
+use Illuminate\Support\Facades\Hash;
+use App\Models\User;
 
 class RoleService
 {
     /**
      * Create Role
      *
      * @param array $requestData
-     * @return array
+     * @return string
      */
-    public function submitRole(array $requestData): array
+    public function submitRole(array $requestData): string
     {
         $name = $requestData['role'];
+        $roles = Role::create(['name' => $name]);
         $checkboxValues = $requestData['checkbox'];
```

Comment:
Please use name that are easily understandable, in this case it should be $rolePermissions and keep the the name singular or plural accordingly.

Reply:
Fixed

Reply:
It was for $checkboxValues, please rename this to $rolePermissions, the one you fixed was OK before.

Reply:
Fixed